pyrasterframes/src/main/python/deps/jars/README.md

```diff
@@ -0,0 +1,4 @@
+# pyrasterframes.jars
+
+Submodule containing JARs needed for pyrasterframe
```

Comment:
I didn't see it in this changeset, but do we need to have SBT put the assembly in here?

Reply:
yep.